lib/elm-test.js

```diff
@@ -332,17 +328,22 @@ if (args._[0] === 'make') {
       args.verbose,
       pathToElmBinary,
       args.report
-    ).then(() => {
-      Runner.getIndirectDeps(projectRootDir).then((packageIndirectDeps) => {
-        return elmTestMake(
-          projectRootDir,
-          pathToElmBinary,
-          testFilePaths,
-          packageIndirectDeps,
-          hasBeenGivenCustomGlobs
-        );
+    )
+      .then(() => {
+        Runner.getIndirectDeps(projectRootDir).then((packageIndirectDeps) => {
+          return elmTestMake(
+            projectRootDir,
+            pathToElmBinary,
+            testFilePaths,
+            packageIndirectDeps,
+            hasBeenGivenCustomGlobs
+          );
+        });
+      })
+      .catch((err) => {
+        console.error(err.message);
+        process.exit(1);
```

Comment:
This used to fail with ”UnhandledPromiseRejectionWarning”. In a couple of other places as well.